private/buf/cmd/buf/command/alpha/repo/reposync/reposync.go

```diff
@@ -79,7 +78,7 @@ func NewCommand(
 
 type flags struct {
 	ErrorFormat      string
-	Modules          []string
+	ModulesDirs      []string
```

Comment:
You could say this is the main trigger for the whole PR. We are no longer receiving module identities overrides, but reading it from HEAD on each branch, for each module directory.